css/properties/animation-play-state.json

```diff
@@ -113,6 +113,74 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "paused": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "3"
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "≤16"
```

Comment:
The introduction of child features here makes this run into the descendent data problem. Looks like you decided to keep things vague with "≤16"? Is that a decision you made throughout?

The actual story here is:
css.properties.animation-play-state.paused: Firefox 16 
css.properties.animation-play-state.paused: Firefox 5 with -moz- prefix

Reply:
See https://github.com/mdn/browser-compat-data/pull/22329#discussion_r1512447140 for more details as to why ≤15 and ≤16 may appear for Firefox; note it has nothing to do with the prefixes.